modules/ossm-about-concepts-argo-rollouts.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-argo-rollouts_{context}"]
+= {SMProductName} and Argo Rollouts
+
+{SMProductName}, when used with Argo Rollouts, provides more advanced routing capabilities by using Istio, and does not require the configuration of a sidecar container.
+
+You can use {SMProduct} to split traffic between two application versions.
+
+* *Canary version*: A new version of an application where you gradually route the traffic.
+* *Stable version*: The current version of an application. After the canary version is stable and has all the user traffic directed to it, it becomes the new stable version. The previous stable version is discarded.
+
+The Istio-support within Argo Rollouts uses the `Gateway` and `VirtualService` resources to handle traffic routing.
+
+* *Gateway*: You can use a Gateway to manage inbound and outbound traffic for your mesh. The gateway is the entry point of {SMProduct} and handles traffic requests sent to an application.
+
+* *VirtualService*: `VirtualService` defines traffic routing rules and the percentage of traffic that goes to underlying services, such as the stable and canary services.
```

Comment:
Maybe make these two start with an article to parallel the previous definitions??

```suggestion
* *Gateway*: The gateway is the entry point of {SMProduct} and handles traffic requests sent to an application. You can use a Gateway to manage inbound and outbound traffic for your mesh. 

* *VirtualService*: The traffic routing rules and the percentage of traffic that goes to underlying services, such as the stable and canary services.
```